prescriptions-gh-release-notes-job/base/openshift-templates/adviser.yaml

```diff
@@ -0,0 +1,101 @@
+spec:
+  serviceAccountName: argo
+  activeDeadlineSeconds: 3000
+  ttlStrategy:
+    secondsAfterCompletion: 7200
+    secondsAfterSuccess: 7200
+    secondsAfterFailure: 7200
+  entrypoint: prescriptions-gh-release-notes
+  volumes:
+    - name: output
+      emptyDir: {}
+  schedule: "@weekly"
+  concurrencyPolicy: "Forbid"
+  startingDeadlineSeconds: null
+  workflowSpec:
+    entrypoint: prescriptions-gh-release-notes
+    templates:
+    - name: prescriptions-gh-release-notes
+      container:
+        image: quay.io/thoth-station/prescriptions-gh-release-notes-job:v0.0.2
+        env:
+          - name: THOTH_LOGGING_NO_JSON
+            value: "1"
+          - name: THOTH_PRESCRIPTIONS_GH_RELEASE_NOTES_DEBUG
+            value: "1"
+          - name: THOHT_PRESCRIPTIONS_GH_RELEASE_NOTES_START_DATE
+          - name: THOHT_PRESCRIPTIONS_GH_RELEASE_NOTES_END_DATE
+          - name: THOHT_PRESCRIPTIONS_GH_RELEASE_NOTES_OUTPUT
+            value: "/mnt/workdir/prescriptions_gh_release_notes.yaml"
+          - name: THOTH_DEPLOYMENT_NAME
+            valueFrom:
+              configMapKeyRef:
+                key: deployment-name
+                name: thoth
+          - name: SENTRY_DSN
+            valueFrom:
+              secretKeyRef:
+                name: thoth
+                key: sentry-dsn
+          - name: THOTH_S3_ENDPOINT_URL
+            valueFrom:
+              configMapKeyRef:
+                key: host
+                name: ceph
+          - name: THOTH_CEPH_BUCKET
+            valueFrom:
+              configMapKeyRef:
+                key: bucket-name
+                name: ceph
+          - name: THOTH_CEPH_BUCKET_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                key: bucket-prefix
+                name: ceph
+          - name: THOTH_CEPH_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                name: ceph
+                key: key-id
+          - name: THOTH_CEPH_SECRET_KEY
+            valueFrom:
+              secretKeyRef:
+                name: ceph
+                key: secret-key
+        # outputs:
+        #   artifacts:
+        #     - name: outputdocument
+        #       path: "/mnt/workdir/prescriptions_gh_release_notes.yaml"
+        #       archive:
+        #         none: {}
+        #       s3:
+        #         key: "{{inputs.parameters.THOTH_CEPH_BUCKET_PREFIX}}/\
+        #           {{inputs.parameters.THOTH_DEPLOYMENT_NAME}}/\
+        #           prescriptions_gh_release_notes.yaml"
+        #         endpoint: "{{inputs.parameters.THOTH_S3_ENDPOINT_URL}}"
+        #         bucket: "{{inputs.parameters.THOTH_CEPH_BUCKET_NAME}}"
+        #         insecure: true
+        #         accessKeySecret:
+        #           name: argo-artifact-repository-secrets
+        #           key: accessKey
+        #         secretKeySecret:
+        #           name: argo-artifact-repository-secrets
+        #           key: secretKey
```

Comment:
We will need to use `resource` here to propagate values from configmap, see https://github.com/argoproj/argo-workflows/issues/3212#issuecomment-642824124